# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/*/glibc/csky.patch.cross0
# Copyright (C) 2021 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- glibc-2.33/sysdeps/csky/preconfigure.vanilla	2021-05-01 14:34:51.164756147 +0200
+++ glibc-2.33/sysdeps/csky/preconfigure	2021-05-01 14:35:43.370756125 +0200
@@ -9,6 +9,8 @@
     hard_float_abi=`$CC $CFLAGS $CPPFLAGS -E -dM -xc /dev/null |
       sed -n 's/^#define __CSKY_HARD_FLOAT_ABI__ \(.*\)/\1/p'`
 
+    abi=2
+
     case "$abi" in
     1)
         echo "glibc does not support abiv1 yet" >&2
